### ISSUE TITLE
livewireChartsColumnChart is not defined

### ISSUE THREAD

**Issue by jhourlad** [asantibanez/livewire-charts#20]
Getting the error: `livewireChartsColumnChart is not defined` even after putting in `@livewireChartsScript` in the scripts section of the layout. The culprit is that composer installs the package into `/vendor/asantibanez/livewire-scripts` while @livewireChartsScripts points it to `/livewire-scripts`.

```
<div>
   <livewire:livewire-column-chart :column-chart-model="$chart" />
   ...
   @push('scripts')
           <script src="//cdnjs.cloudflare.com/ajax/libs/apexcharts/3.22.2/apexcharts.min.js" integrity="sha512-MaTc94UwSWnpm+S76wtgEHwUsQa3P8ed8iqgCmVHJMxfrDdF0/+2Ji/T7wAE4UZqIEapiG+g9kPaKZ6bMJtuPg==" crossorigin="anonymous"></script>
           <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/apexcharts/3.22.2/apexcharts.min.css" integrity="sha512-H6KANMpZQKsK9c09UqdcQv02JTiZ/hMVwxkcbDLrp125CR884wwEdnWDm+Yuo6tihuC/cizLYWAjMZi0xAfGig==" crossorigin="anonymous" />
           @livewireChartsScripts
   @endpush
</div>
```
Resulting HTML codes show livewire-charts js file is indeed appended to the codes:
![image](https://user-images.githubusercontent.com/3771945/101158689-15310e80-3667-11eb-8766-1f77fdcc390c.png)

**Comment by jhourlad** [comment]
Temporary fix I am using right now is not to use `@livewireChartsScripts` 
then manually linking to the plugin's script:

 `<script src="/vendor/asantibanez/livewire-charts/public/app.js"></script>`

**Comment by linusx** [comment]
Did you run this command: `php artisan livewire:publish --assets`

**Comment by jhourlad** [comment]
Are we intentionally hiding that from the documentation?


*JHOURLAD G ESTRELLA*
Email: jhourlad01@gmail.com
Mobile: +63 (916) 1-888-535

**Comment by linusx** [comment]
I think I found it in the docs somewhere after having similar issues. Just not right up front.

**Comment by asantibanez** [comment]
Hello everyone! Thanks for checking out the package.

I've added a build system so we can compile all js into a single app.js file and reference that via the `@livewireChartsScripts` directive.

I've added in the docs this step

![image](https://user-images.githubusercontent.com/5126648/101211842-cb83fc00-3645-11eb-946b-6f6c5d2762fd.png)

I'll edit this instructions to make them clearer.

Let me know if this fixes the issue.
